submissions/2020-12-18-OpenFF-Theory-Benchmarking-Set-v1.0/compute.json

```diff
@@ -0,0 +1,95 @@
+{
+  "qc_specifications": {
+    "WB97X-D3BJ": {
```

Comment:
Should we name this one `"WB97X-D3BJ/DZVP"` (also down in `spec_name`)?

Reply:
Thanks for the review, updated the fields to be consistent with other submissions.